docs/team/siqiang-ng.md

```diff
@@ -3,79 +3,68 @@ layout: page
 title: Ng Si Qiang's Project Portfolio Page
 ---
 
-## Project: Projact
-
-Projact is a **project management app that helps NUS computing students to organise their fellow computing students' contacts and their teams' meeting links and tasks**. It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI).
+### Project: Projact 
+<hr>
+Projact is a contact and project management app that helps NUS computing students to
+ organise their contacts and their projects' meeting links and tasks. 
 
 Given below are my contributions to the project.
```

Comment:
i think it's nicer if there's a space between Projact's description and given below...? but if there's really no space then nvm haha

Reply:
Yep there is a space i thought? HAHA

Reply:
ok i realise what u mean, made the changes u can see